source/adapters/hip/enqueue.cpp

```diff
@@ -17,11 +18,9 @@
 
 extern size_t imageElementByteSize(hipArray_Format ArrayFormat);
 
-namespace {
-
-ur_result_t enqueueEventsWait(ur_queue_handle_t, hipStream_t Stream,
-                              uint32_t NumEventsInWaitList,
-                              const ur_event_handle_t *EventWaitList) {
+UR_APIEXPORT ur_result_t UR_APICALL enqueueEventsWait(
```

Comment:
I don't think we should have `UR_APIEXPORT ` and `UR_APICALL ` here, as this isn't a UR spec defined entry-point. Same for `simpleGuessLocalWorkSize`

Reply:
You're right. This file has been updated accordingly.